open_humans/templates/email/welcome.html

```diff
@@ -1,10 +1,8 @@
 {% load utilities %}<p><strong>Thank you for joining the Open Humans community!</strong></p>
 
 <p>
-  It's our mission to help connect you and your data with researchers.
-  In Open Humans, you'll be able to build a profile with data you receive
-  from studies and commercial sources. We'll invite you to share that data
-  in various ways, publicly or directly with projects.
+  It's our mission to empower you with your personal data, to explore and
```

Comment:
i think this could be a little clearer; the comma after personal data trips me up (what am i exploring and sharing?)

Reply:
Fair point. But... I think I'd like to leave it vague for now. Hoping the board will be able to discuss more at our next meeting, we may be moving in directions that aren't requiring sharing data itself (i.e. sharing in other ways: tools, notebooks, other collaborating).